pkgs/applications/qubes/qubes-core-vchan-xen/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, xen_4_10 }:
+
+stdenv.mkDerivation rec {
+  pname = "qubes-core-vchan-xen";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "QubesOS";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1p7gc7lyi53zcy9dyikjrkkmfa9873sabypqb7cp4cr4371cl8q6";
+  };
+
+  buildInputs = [ xen_4_10 ];
+  buildPhase = ''
+    make all
+  '';
+  installPhase = ''
+    make install DESTDIR=$out
```

```suggestion
    make install DESTDIR=$out PREFIX=/
```
There is also LIBDIR and INCLUDEDIR if needed.